core/globals.h

```diff
@@ -308,6 +308,17 @@ typedef struct _dr_stats_t {
     uint64 synchs_not_at_safe_spot;
 } dr_stats_t;
 
+/**
+ * Error codes of DR API routines.
+ */
+typedef enum {
+    DR_ERROR_NONE,
+    DR_ERROR_INVALID_PARAM,
```

Comment:
suggest: Spell out PARAMETER. That seems nicer IMHO, but more importantly it will match drreg, drsyms, drcovlib, and drbbdup.

Reply:
Done.